config.py

```diff
@@ -0,0 +1,40 @@
+import dotenv
+import os
+import urllib.parse as urlparse
+
+if os.path.isfile(".env"):
+    dot_env_conf = dotenv.load_dotenv(".env")
+
+SECRET_KEY = os.environ.get("SECRET_KEY")
+
+url = urlparse.urlparse(os.environ["DATABASE_URL"])  # os.environ['DATABASE_URL']
+dbname = url.path[1:]
+user = url.username
+password = url.password
+host = url.hostname
+port = url.port
+
+DATABASES = {
+    "default": {
+        "ENGINE": "django.db.backends.postgresql",  # your database
+        "NAME": str(dbname),
+        "USER": user,
+        "PASSWORD": password,
+        "HOST": host,
+        "PORT": port,
+    }
+}
+
+ALLOWED_HOSTS = [os.environ.get("ALLOWED_HOST")]
+
+# Discord stuff
+TOKEN: str = os.environ.get("TOKEN")  # > https://discord.com/developers/applications
+LOG_WEBHOOK = os.environ.get(
+    "LOG_WEBHOOK"
+)  # create webhook in the server for a channel and paste here
```

Comment:
Put info about config values in the readme instead

Reply:
feel free to do it :D